UPGRADE-15.0.md

```diff
@@ -863,6 +863,7 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 
 #### remove product stock from calculated sale exclusion calculation ([#3159](https://github.com/shopsys/shopsys/pull/3159))
 
+-   all the add to cart buttons are now disabled when product quantity is zero or negative
```

Comment:
this sentence does not need to be in the BE section but I can live with that :slightly_smiling_face: